### ISSUE TITLE
Email notifications missing submission object during send or when manually re-sending

### ISSUE THREAD

**Issue by bliswebagency** [solspace/craft3-freeform#1385]
### What happened?

When sending email notifications for our stored form submission, the submission object appears to be empty or missing from the notification template. This occurs whether during the submission process or if when we manually re-send from the dashboard.

However, if we choose 'Send Additional Notification' the submission object is accessible in the template and content is displayed, as per this sample template:

```
<p> This is blank for initial or re-sent notifications: {{ submission.id }} - {{ submission.token }} </p>
<p> But this works with 'Send Additional Notification' {{ submission.id }} - {{ submission.token }} </p>
```

### How can we reproduce this?

1. Create an email notification template that outputs the submission.id
2. Make sure the form is storing the submissions
3. Submit form and check email (blank submission.id)
4. Re-send notification and check email (blank submission.id)
5. Choose 'Send additional notification' and check email (submission.id is present)

### Freeform Edition

Pro

### Freeform Version

5.5.1

### Craft Version

4.10.3

### When did this issue start?

Unsure

**Comment by StuartMcD** [comment]
Can also confirm same behaviour.

**Comment by gustavs-gutmanis** [comment]
Hi @bliswebagency, @StuartMcD

Thank you for reporting this, the issue has been found and rectified. We'll release a fix for this shortly.

**Comment by bliswebagency** [comment]
Hi @gustavs-gutmanis and @kjmartens , thank you for looking into this.

Would it be advisable to wait for the next updated release to apply this fix #1385, or could we safely apply it to the source in the interim?

**Comment by kjmartens** [comment]
Hi @bliswebagency and @StuartMcD,

This has now been resolved in Freeform 5.5.2. 🙂
